src/Imgproxy/V1/Fit.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neighborhoods\ImgProxyClientComponent\Imgproxy\V1;
+
+/**
+ * @link https://docs.imgproxy.net/#/generating_the_url_basic?id=resizing-types
+ *
+ * imgproxy supports the following resizing types
+ */
+final class Fit
```

Comment:
i may be misreading the docs, but it feels like the class name should be `ResizingType`, no?

Reply:
That's what ImgProxy™ calls it, but this component uses it for the `setFit()` method.